packages/forms/src/model/form_group.ts

```diff
@@ -629,49 +625,53 @@ export class FormRecord<TControl extends AbstractControl<Value<TControl>, RawVal
  *
  * @publicApi
  */
+export class FormRecord<TControl extends AbstractControl<ɵValue<TControl>, ɵRawValue<TControl>> =
+                                             AbstractControl> extends
+    FormGroup<{[key: string]: TControl}> {}
+
 export interface FormRecord<TControl> {
   /**
    * Registers a control with the records's list of controls.
    *
-   * {@see FormGroup#registerControl}
+   * See `FormGroup#registerControl`
    */
   registerControl(name: string, control: TControl): TControl;
 
   /**
    * Add a control to this group.
    *
-   * {@see FormGroup#addControl}
+   * See `FormGroup#addControl`
    */
   addControl(name: string, control: TControl, options?: {emitEvent?: boolean}): void;
 
   /**
    * Remove a control from this group.
    *
-   * {@see FormGroup#removeControl}
+   * See `FormGroup#removeControl`
    */
   removeControl(name: string, options?: {emitEvent?: boolean}): void;
 
   /**
    * Replace an existing control.
    *
-   * {@see FormGroup#setControl}
+   * See `FormGroup#setControl`
```

Comment:
nit: I'd propose updating this line (and similar ones in this PR) a bit:
```suggestion
   * See `FormGroup#setControl` for additional information.
```